lib/axe-injector.js

```diff
@@ -9,15 +11,20 @@ class AxeInjector {
   }
 
   // Single-shot error handler. Ensures we don't log more than once.
-  errorHandler() {
+  errorHandler(err) {
     // We've already "warned" the user. No need to do it again (mostly for backwards compatiability)
     if (this.didLogError) {
       return;
     }
 
     this.didLogError = true;
-    // eslint-disable-next-line no-console
-    console.error('Failed to inject axe-core into one of the iframes!');
+    if (err instanceof StaleElementReferenceError) {
+      // eslint-disable-next-line no-console
+      console.error('Tried to inject into a removed iframe. Please ensure the page has settled.')
```

Comment:
Can we change this to: `Tried to inject into a removed iframe. This will not affect the analysis of the rest of the page but you might want to ensure the page has finished updating before starting the analysis`